product_harmonized_system/migrations/14.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, api
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        pc_field_id = env.ref(
+            "product_harmonized_system.field_product_category__hs_code_id"
+        ).id
+        cr.execute(
+            """
+            UPDATE product_category pc
+            SET hs_code_id=SUBSTRING(ip.value_reference, 9, 99)::int
+            FROM ir_property ip
+            WHERE ip.res_id like 'product.category,%%' AND
+            SUBSTRING(ip.res_id, 18, 99)::int=pc.id AND
+            ip.name='hs_code_id' AND
+            ip.company_id IS NOT null AND
```

Comment:
I don't think you should restrict this. Maybe there's a generic value for all companies in the DB. ir.property allows it...

Reply:
done